proxy/src/http/org.rs

```diff
@@ -702,12 +702,12 @@ mod test {
                     "name": project_name.to_string(),
                 },
                 "registration": Value::Null,
-                "shareableEntityIdentifier": urn.to_string(),
+                "shareableEntityIdentifier": format!("%{}", urn),
                 "stats": {
-                    "branches": 11,
-                    "commits": 267,
-                    "contributors": 8,
-                },
+                  "branches": 2,
```

Comment:
An issue we have is that the `surf::Stats` has `*Count` as the names. We should either update here, or we update `surf`.

Reply:
See https://github.com/radicle-dev/radicle-surf/pull/145; will monitor https://github.com/radicle-dev/radicle-link/pull/112 to figure out the best time for bumping the surf version

Reply:
FYI: #112 isn't the issue. It's what's currently on master that I think could cause issues. You could try and see by picking the head commit. Maybe it'd be good to see if you can fix anything that shows up 😁 I'm also just speculating that there's compat issues, I haven't confirmed personally

Reply:
Ok I've confirmed the errors 😄 they look like: `method not found in &librad::net::peer::PeerApi`. Will continue digging into them on Monday

Reply:
I'm fixing this in another branch :) Sorry you've gotten caught between a rock and a hard place currently. But we should be able to avoid this if https://github.com/radicle-dev/radicle-upstream/pull/609 is accepted :+1: